src/Client.ts

```diff
@@ -3,8 +3,9 @@ import {
   GatewayIntentBits,
   GatewayOpcodes
 } from 'discord-api-types/v9'
-import * as D from 'discord.js'
-import WebSocketShard from '../node_modules/discord.js/src/client/websocket/WebSocketShard'
+import * as Discord from 'discord.js'
+var {WebSocketShard} = require('discord.js')
```

Comment:
`import { WebSockectShard } from ‘discord.js’` should work here. Let’s keep imports/exports consistent.